notification-portlet-webapp/src/main/webapp/scripts/modal.js

```diff
@@ -0,0 +1,128 @@
+Vue.component("modal", {
+  template: "#modal-template",
+  props: {
+    id: String
+  }
+})
+
+// start app
+new Vue({
+  el: "#app",
```

Comment:
From @cousquer on Slack

> just a simple question: what will happen if there is in the DOM another portlet with `<div id="app">` ? :thinking: 
>
> answer: the first one will win, the second will throw an error...

Reply:
in fact, I'm not sure, it's a real question... 😃

Reply:
It will render to the first id of `app` it sees. The other id will be empty. It has to be more unique. Thanks @cousquer for pointing this out :)